projects/ngx-charts-on-fhir/src/lib/data-layer-list/annotation-options/annotation-options.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import produce from 'immer';
+import { merge } from 'lodash-es';
+import { DataLayerColorService } from '../../data-layer/data-layer-color.service';
+
+@Component({
+  selector: 'annotation-options',
+  templateUrl: './annotation-options.component.html',
+  styleUrls: ['./annotation-options.component.css'],
+})
+export class AnnotationOptionsComponent implements OnInit {
+  private _annotation?: any;
+  newAnnotation?: any;
+
+  @Input() set annotation(annotation: any) {
+    this._annotation = annotation;
+    this.updateForm(annotation);
+  }
+
+  @Output() annotationsChange = new EventEmitter<any>();
```

Comment:
This is now the exact same event name as annotation-list component and we could run into similar issue with event name collisions. It's also a bit misleading because the output name is plural but it's only emitting a single annotation. Can you please change this to `annotationChange` (singular)?